charm_versions.md

```diff
@@ -0,0 +1,138 @@
+# Versions
```

Comment:
Hi Tony, thanks for preparing this. My first question: Where would you like to have this published? I'm asking because we already have some of this content in the docs.